src/aics-image-viewer/components/AxisClipSliders/index.tsx

```diff
@@ -41,22 +41,25 @@ const SliderRow: React.FC<SliderRowProps> = ({
   onEnd,
 }) => {
   const isRange = vals.length > 1;
-  // If slider is a range, handles represent slice *edges*: the range around only the max slice is [max-1, max], e.g.
-  // If slider is not a range, handle represents 0-indexed slices: last slice is at max-1
-  const inputMax = isRange ? max : max - 1;
+  // If slider is a range, handles represent slice *edges*: the range around only the 1st slice is [0, 1]; the range
+  // around only the last is [max-1, max].
```

Comment:
Maybe it would be clearer to say that the bounds are `[min, max)` or `inclusive min, exclusive max`?